mgmtsystem_nonconformity/reports/mgmtsystem_nonconformity_report.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+    <!-- Management System Non Conformity Report Pivot View -->
+
+
+    <!-- Non Conformity Report Graph View -->
+
+    <!-- Management system Non Conformity Report Action -->
+
+    <record id="action_report_mgmtsystem_nonconformity" model="ir.actions.act_window">
+        <field name="name">Management System Analysis</field>
+        <field name="res_model">mgmtsystem.action.report</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">pivot,graph</field>
+        <field name="context">{"search_default_year":1,"search_default_User":1,"search_default_This Month":1,'group_by_no_leaf':1,'group_by':[]}</field>
```

Comment:
I choose a wrong branches when i started to work on it. And i don't realise that i haven't revert the changes